crates/cxx-qt-gen/src/parser/cxxqtdata.rs

```diff
@@ -217,9 +220,14 @@ impl ParsedCxxQtData {
                     //
                     // Note that we need to test for qsignal first as qsignals have their own inherit meaning
                 } else if attribute_take_path(&mut foreign_fn.attrs, &["inherit"]).is_some() {
+                    let parsed_inherited_method_self =
+                        ParsedInheritedMethod::parse(foreign_fn.clone(), safe_call)?;
+
                     let parsed_inherited_method =
                         ParsedInheritedMethod::parse(foreign_fn, safe_call)?;
 
+                    self.inherited_methods.push(parsed_inherited_method_self);
+
```

Comment:
why do we have the same thing twice?  why not this

```rust
let parsed_inherited_method = ParsedInheritedMethod::parse(foreign_fn, safe_call)?;
self.inherited_methods.push(parsed_inherited_method.clone());

self.with_qobject(&parsed_inherited_method.qobject_ident)?...
```

Reply:
so that it adds it to the methods contained in cxxqtdata but doesn't break the rest of the code, which is all still using the old method (same way we did things for methods and signals, where I implemented it all before removing the old code)

Reply:
OK, it gets removed in a later branch anyway, so lets go with it.